src/languages/en.ts

```diff
@@ -1302,6 +1302,9 @@ export default {
             },
         },
     },
+    reportDetailsPage: {
+        in: 'in',
+    },
```

Comment:
shall we use the function pattern instead?

i.e.

```
            inWorkspace: ({policyName}) =>
                `in ${policyName}.`,
```

Reply:
Good catch, I'll change it to a function pattern here.